installer/helpers.sh

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+OPNAMESPACE="porter-operator-system"
+
+setCustomNamespaceForOperator() {
+  if [ -z "$1" ]; then
+    echo "No namespace specified, using default $OPNAMESPACE"
```

Comment:
Let's always be clear which namespace we are referring to in these messages, because we have both the "porter operator namespace" and also a namespace that is created and configured by the `configureNamespace` custom action in the bundle.

```suggestion
    echo "No porter operator namespace specified, using default $OPNAMESPACE"
```